misc fixes (#1180)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/callout-with-media.mdx

| ---------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | -------- | ----------- | ----------------------------------------- |
| Headline                                                                                                                                                   | Short copy | Yes      | 1           | XXS (40 / 55)                             |
| Paragraph                                                                                                                                                  | Long copy  | No       | xe 1        | XXL (350 / 450)                           |
| [Video](https://www.ibm.com/standards/carbon/components/video) or [Image with caption](https://www.ibm.com/standards/carbon/components/image-with-caption) | Component  | Yes      | 1           | –                                         |

> To see the list of all character count standards, go to this

Reviewer: ```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

> <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.

Reviewer: ```suggestion
```
